helios/management/commands/makeboothmessages.py

```diff
@@ -173,16 +171,15 @@ def process_file(file, dirpath, potfile, domain, verbosity,
                 return
         is_templatized = True
         orig_file = os.path.join(dirpath, file)
-        src_data = open(orig_file).read()
+        with open(orig_file) as f:
+            src_data = f.read()
         # MOD
         #src_data = prepare_js_for_gettext(src_data)
         thefile = '%s.c' % file
         work_file = os.path.join(dirpath, thefile)
         f = open(work_file, "w")
```

Comment:
I still don't see that line used anywhere. Maybe we could remove it?
@pwmarcz @szpone ?

Reply:
Isn't it used a couple of lines down when another file is opened and src_data is being written in it?

Reply:
Just checked it and it doesn't look like it's used. All other occurrences in `process_file` are used with `with`.
It looks like it's a duplicate of next line:
`f = open(work_file, "w")`
`with open(work_file, "w") as f:`
They both are opening the same file in the same way and one is not used.